scripts/postprovision.sh

```diff
@@ -0,0 +1,30 @@
+ #!/bin/sh
+
+if ! command -v python -v &> /dev/null
+then
+    echo 'Python not found on $PATH'
+    exit -1
+fi
+
+echo 'Creating python virtual environment "scripts/.venv"'
+python -m venv scripts/.venv
+
+
+echo 'Installing dependencies from "requirements.txt" into virtual environment'
+./scripts/.venv/bin/python -m pip install -r scripts/requirements.txt
+
+if [ "$AZURE_STORATE_ACCOUNT" == "" ]
+then
+    echo 'Environment variables not set - this script assumes that it runs as part of an azd postprovision hook.'
+    echo 'You can run the script manually by passing in the appropriate command line parameters after'
+    echo 'activting the scripts/.venv virtual environment.'
+    echo 'For more information about virtual environments, see https://docs.python.org/3/tutorial/venv.html'
+    echo
+    exit -1
+fi
```

Comment:
I load them automatically so it can be run outside of a hook and not ask the user to load, see the ps1 version.

Reply:
We also pip install dependencies in the script ps1 version. It's a bit of Russian dolls at the moment with cli calling a hook script calling a init script - not to mention and the hook script has dual purpose (restore execution environment + execute init script). This is why I tried to use `azd restore` in #11.  

I think it would be better to load environment variables from `azd` in the `prepdocs.py` script than wrapping `prepdocs.py` script in another script just to set environment variables. But that is a bigger change. I intended to follow up with a clean-up of the `prepdocs.py` script (I ran across a few bugs in there while doing this work), but I wanted to stop the pollution of the global environment first.

Reply:
I would rather take the time now to do the best option, even if a bigger change.